blockchain/README.md

```diff
@@ -1,7 +1,44 @@
 # Blockchain
 
-A Blockchain is a type of distributed ledger technology (DLT) that consists of growing list of records, called blocks, that are securely linked together using cryptography.
+A Blockchain is a type of distributed ledger technology **(DLT)** that consists of growing list of records, called **blocks**, that are securely linked together using **cryptography**.
 
+Let's breakdown the terminologies in the above definition. We find below terminologies,
+
+- Digital Ledger Technology(DLT)
+- Blocks
+- Cryptography
+
+## Digital Ledger Technology
+
+ It is otherwise called as distributed ledger technology. It is simply the opposite of centralized database. Firstly, what is a **ledger**? A ledger is a book or collection of accounts that records account transactions.
+
+ *Then why is Blockchain addressed as digital ledger when it can record more than account transactions?* *And what are other transactions and details it can hold?*
+
+Digital Ledger Technology is just a ledger which is shared among multiple nodes. This way there exist no need for central authority to hold the info. Okay, how is it differentiated from central database and what are their benefits?
+
+There is an organization which has 4 branches whose data are stored in a centralized database. So even if one branch needs any data from ledger they need an approval from database in charge. And if one hacks the central database he gets to tamper and control all the data.
+
+Now lets assume every branch has a copy of the ledger and then once anything is added to the ledger by anyone branch it is gonna automatically reflect in all other ledgers available in other branch. This is done using Peer-to-peer network.
+
+So this means even if tampered in one branch we can find out. If one branch is hacked we can be alerted and get to safeguard other branch. Now, assume these branches as computer or nodes and the ledger is a transaction record or digital receipt. so this is the concept of Digital Ledger Technology.
+
+*Do all node needs all the info in others nodes? Does this increase storage space in all nodes?*
+
+## Blocks
+
+In short a block is nothing but collections of records with a labelled header. These are connected cryptographically. Once a new block is added to a chain previous block is connected or precisely said as locked and will remain unaltered. we will see how this works once we dig deep into working mechanisms.
```

Comment:
```suggestion
In short a block is nothing but collections of records with a labelled header. These are connected cryptographically. Once a new block is added to a chain previous block is connected or precisely said as locked and will remain unaltered. We will see how this works once we dig deep into working mechanisms.
```

Reply:
Hi, Thanks for the feedback. As per above suggestions, I did changes to the document and committed it to this PR. Please revert back if any other changes are required in the document.